roles/build-release/tasks/setup.yaml

```
@@ -17,3 +17,9 @@
     force: true
   # By setting antsibull_data_reset to false, we can run with an edited or otherwise prepared version of ansible-build-data
   when: antsibull_data_reset | bool
+
+- name: Ensure {{ antsibull_data_git_dir }} exists when antsibull_data_reset variable is set to false
+  ansible.builtin.stat:
+    path: "{{ antsibull_data_git_dir }}"
+  register: antsibull_data_directory
+  failed_when: antsibull_data_reset == 0 and antsibull_data_directory.stat.exists == 0
```

How about only executing this when `not antsibull_data_reset`:
```suggestion
  failed_when: antsibull_data_directory.stat.exists == 0
  when: antsibull_data_reset == 0
```
Also, I think we should work with booleans, not with integers:
How about only executing this when `not antsibull_data_reset`:
```suggestion
  failed_when: not antsibull_data_directory.stat.exists
  when: not (antsibull_data_reset | bool)
```

---

Perfect! I committed the suggestions. Thanks @felixfontein:)